vllm/model_executor/weight_utils.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, *args, **kwargs):
 
 
 def get_lock(model_name_or_path: str, cache_dir: Optional[str] = None):
-    lock_dir = cache_dir if cache_dir is not None else "/tmp"
+    lock_dir = cache_dir if cache_dir is not None else "~/vllm_cache"
```

Comment:
Nit: lets remove this

Reply:
thanks for catching this, I didn't notice I left it here